pkgs/by-name/li/libvmi/package.nix

```
@@ -0,0 +1,91 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  autoconf-archive,
+  bison,
+  flex,
+  glib,
+  pkg-config,
+  json_c,
+  libvirt,
+
+  withVMIFS ? true,
+  fuse,
+
+  legacyKVM ? false,
+  libkvmi,
+
+  xenSupport ? true,
+  xen-slim,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libvmi";
+  version = "0.14.0-unstable-2024-07-25";
+  libVersion = "0.0.15";
+
+  outputs = [
+    "out" # Contains binaries helpful for testing the libraries. Must be run on the host system.
+    "lib" # Contains the LibVMI libraries. This is most likely what you want.
+    "dev" # Development headers and the pkg-config spec.
+  ];
```

```suggestion
  outputs = [ "out" "lib" "dev" ];
```
that is well known and documented in the stdenv docs

---

Very well.